structure/0006-governance.md

```diff
@@ -2,53 +2,161 @@
+## The Administrative Board
 
-If there are less than three members on the administrative board, each remaining board member may nominate a community member (potentially a non-staff member) for the position. Existing staff members may nominate themselves. The remaining staff members may then vote on the nominees, with administrative board member votes being worth three votes only in the event of a tie.
+The administrative board is a group of organization members responsible for tasks that are not assigned to another team.
+It is made up of at least three members to prevent deadlocks in the decision-making process.
+Members of the administrative board may be members of other teams.
 
-### Proposing a Team
-When proposing a new team, an RFC specifying several things needs to be written. Such a proposal should generally follow this structure:
+If it is decided that more administrative board members are required, either due to a current member stepping down, or due to it being decided that the admin board needs to be expanded, an election must take place.
 
-#### Motivation
-Why is this team needed? What current problems will this team solve? What existing projects or systems will this team be responsible for?
+### Admin board Voting Process
 
-#### Explanation
-What will this team *do*? What new projects will be created that this team will be responsible for and how will each of these projects benefit the QuiltMC organization/community? What processes will this team follow for each of the different kinds of changes their projects may undergo?
+ Due to the fact that multiple candidates may be nominated for the admin position when a vote is held, the fact that there may be multiple spots to fill on the board, and the fact that the decision affects the entire organization, the standard voting procedure cannot be used. Instead, the following procedure will be employed:
 
-#### Drawbacks
-Can usually be omitted. Is only necessary if there are clear drawbacks to such a teams existence. Drawbacks may include overlap of responsibilities, excessive granularity, etc.
+Before the vote begins, a group of people, henceforth referred to as the Election Oversight Board or EOB, must be selected to oversee the voting process. They will be responsible for receiving candidate nominations, concerns about nominated candidates from other organization members, and hosting the election. In normal circumstances, the EOB consists of the existing members of the admin board, but, if this is not possible for any reason, the EOB will instead consist of members with the Community Manager role in the Community Team. The initial EOB may add any members that they feel would be a valuable addition to the board, with a simple majority vote.
```

Comment:
Is there a minimum number of people required to form an Election Oversight Board? Should there be such an enforced minimum, akin to the minimum number of members enforced on the administrative board itself?

Perhaps, as a modicum of accountability and transparency, it should be required that at least one member of the EOB not be an existing member of the administrative board.

Reply:
> As I understand it, the EOB already doesn't require members added later to be from any particular affiliation?

Correct